src/main/java/graphql/execution/FieldCollectorParameters.java

```diff
@@ -26,15 +26,15 @@ public GraphQLSchema getGraphQLSchema() {
         return variables;
     }
 
-    public GraphQLObjectType getObjectType() {
-        return objectType;
+    public GraphQLFieldsContainer getFieldsContainer() {
+        return fieldsContainer;
     }
```

Comment:
@andimarek - graphql.schema.DataFetcherSelectionTest

The queries used are of an interface type (eg Character) and as they are executed, the interface type needs to be examined for "fields"